Assets/Scripts/CommandPattern/CommandHandler.cs

```diff
@@ -1,11 +0,0 @@
-﻿using System.Collections;
-using System.Collections.Generic;
-using UnityEngine;
-
-public class CommandHandler : ICommand
-{
-    public void ScreenInteraction(ShootRaycast shootRaycast, IInteractionController interactionController)
```

Comment:
With any type of injection (in the constructor for a pure class or in Init for MB, you should always expect interfaces).
You inject implementations of course but you don't expect them.
For building the implementations I normally use a factory but you can have even a static Instantiator